packages/accordion/README.md

```diff
@@ -0,0 +1,11 @@
+# `yoopta-accordion`
+
+> TODO: description
```

Comment:
Wondering if README is mandatory file for publishing the plugin to npm, it's a bit misleading.